src/pages/home/_hero.js

```diff
@@ -41,6 +48,56 @@ export const Hero = () => {
         return errors
     }
 
+    const getVerifyEmailRequest = email => {
+        const utm_data = TrafficSource.getData()
+        const affiliate_token = Cookies.getJSON('affiliate_tracking')
```

Comment:
Why did you use Cookies instead of CookieStorage to get the cookie?

Reply:
affiliate_tracking used CookieStorage at TrafficSource to write it, but, to get it, i want to extract the whole cookie value as an object